LayoutTests/fast/forms/date-multiple-fields/date-multiple-fields-keyboard-events.html

```diff
@@ -0,0 +1,258 @@
+<!DOCTYPE html>
+ <html>
+ <head>
+ <script src="../../../resources/js-test.js"></script>
+ </head>
+ <body>
+ <p id="description"></p>
+ <p>
+ Please run this with DumpRenderTree.
+ </p>
+ Test following keys:
+ <ul>
+     <li>Digits</li>
+     <li>Left/Right - Move focus field inside element</li>
+     <li>Up/Down - Increment/decrement value of focus field</li>
+     <li>Tab - Move focus field</li>
+     <li>Backspace - Make value empty</li>
+ </ul>
+ <input id="before">
+ <input id="input" type="date">
+ <input id="after">
+ <div id="console"></div>
+ <script>
+ description('Multiple fields UI of month input type with keyboard events');
+ if (window.internals)
+     internals.settings.setLangAttributeAwareFormControlUIEnabled(true);
```

Comment:
We don't have this testing function in WebKit. Not entirely sure what it does, but I could see how date control tests could depend on locale.

Reply:
No, I'm wrong, it's just generated code. Looks like we do support this.

Reply:
Right. You can either extract subtests that yield same result everywhere or you can add different expected result for each platform. Go check out https://github.com/WebKit/WebKit/blob/main/Introduction.md#layout-tests-tests-of-the-web-for-the-web